doc/security_issue_runbook.md

```diff
@@ -23,7 +23,7 @@ For information on what types of issues are considered security vulnerabilities
 
 - Finalize Details: Once a fix is developed and the vulnerability is fully understood, finalize the advisory details and prepare for public release. Ensure the security issues are resolved in the private fork.
 - Request CVE: Use the Big Green Button on the advisory to request a CVE number from GitHub staff.
-- Advanced Disclosure Email: Decide on a disclosure date, typically within a week, and send an email to sec-announce@bytecodealliance.org about the upcoming security release. Other ways are also available to communicate the disclosure date.
+- Advanced Disclosure Email: Decide on a disclosure date, typically within a week, and send an email to sec-announce@bytecodealliance.org about the upcoming security release. It can be accomplished by posting on https://groups.google.com/a/bytecodealliance.org/g/sec-announce. Other ways are also available to communicate the disclosure date.
```

Comment:
are you just suggesting an alternative way to post to the ML?

or is it recommended to use the web interface?

Reply:
it is recommended to use the web interface

Reply:
why?
anyway, in that case, it's better to make it clear i guess.
eg. `We recommend to use the web interface on ....`

Reply:
It is a kind of BA requests ...
<img width="812" height="599" alt="image" src="https://github.com/user-attachments/assets/93af2e8b-1f07-4e0d-abad-23cb3ff55ce4" />

Reply:
> It is a kind of BA requests ...

BA requested to use the web interface instead of emails?
why?